src/message/MessageContainer.js

```diff
@@ -16,29 +18,60 @@ const htmlToDomTree = html => {
   return domTree;
 };
 
-export default class MessageContainer extends React.PureComponent {
+
+class MessageContainer extends React.PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      showActionSheet: false
+    };
+  }
 
   handlePress = () => {
+    const isFromReply = false;
+    this.narrow(isFromReply);
+  };
+
+  narrow = (isFromReply: boolean) => {
     const { message, doNarrow } = this.props;
-    doNarrow(narrowFromMessage(message), message.id);
-  }
+    doNarrow(narrowFromMessage(message), message.id, isFromReply);
+  };
+
+  narrowToConversation = () => {
+    const isFromReply = false;
+    this.narrow(isFromReply);
```

Comment:
You want to make it obvious what the parameter means?
I think it is an overkill and just looks weird.